packages/shell-api/src/helpers.ts

```diff
@@ -203,6 +203,14 @@ export function processDigestPassword(
   return { digestPassword: true };
 }
 
+const SHARDING_VERSION_FIELD_EXCLUSION = {
+  minCompatibleVersion: 0,
+  currentVersion: 0,
+  excluding: 0,
+  upgradeId: 0,
+  upgradeState: 0,
+};
```

Comment:
Totally optional, but I think you can just inline this into the place where it’s being used